src/ai/backend/client/cli/types.py

```diff
@@ -20,3 +20,10 @@ class CLIContext:
     api_config: APIConfig = attr.field()
     output_mode: OutputMode = attr.field()
     output: BaseOutputHandler = attr.field(default=None)
+
+
+class ExitCode(enum.Enum):
+    OK = 0
+    ERROR = 1
+    TIMEOUT = 2
+    INVALID_ARGUMENTS = 3
```

Comment:
Um, I'd like to put this on `ai.backend.cli.types`, not `ai.backend.client.cli.types` because it could be reused across all other CLI modules, even in manager/agent/etc.

Also, how about renaming `ERROR` to `FAILURE` becaues 'error' means a narrower scope of failures such as bug or system failures while 'failure' generally means "we could not accomplish what you wanted" for whatever reasons.

Others look good.

Reply:
I was worried because there were behaviors that were fail rather than errors, but I think it's good that both can be included if it is marked as `failure`.